### PR TITLE
docs(datetime): add title playgrounds

### PR DESCRIPTION
**Pull request by liamdebeasi**
This PR adds the following playgrounds:

1. How to show the default title.
2. How to customize the title text.

These fall under the "Titles" category.

### CODE REVIEW

**Review by sean-perkins — APPROVED** [review]
Changes look good 👍  does this identify a new bug? The hover cursor style on the title contents seems incorrect (it's hinting that you can enter input into the title).

File: docs/api/datetime.md [hunk 1]
```diff
@@ -211,15 +215,15 @@ This example shows a datetime with the `date` configuration.
 
 ## Titles
```
Comment (amandaejohnston) [inline docs/api/datetime.md:216]: Both datetimes have the bottom cut off on `md` mode: https://i.gyazo.com/9a18a81166091c4bffb68ed2f6bbdc86.png

**Comment by liamdebeasi** [comment]
@sean-perkins Where are you seeing that hint?

**Comment by sean-perkins** [comment]
Having trouble getting the cursor to show in a screenshot, here's a video: 

<video src="https://user-images.githubusercontent.com/13732623/166304860-0a4b811b-5c99-438a-8037-8cc06d95782b.mp4"></video>

**Comment by liamdebeasi** [comment]
That cursor is supposed to show for any text that can be selected. For example, you should see the same results if you hover over this message in GitHub. See the "text" type here: https://developer.mozilla.org/en-US/docs/Web/CSS/cursor#values

**Review by amandaejohnston — APPROVED** [review]

**Pull request merged**